sdk/python/sentience/cli.py

```diff
@@ -0,0 +1,359 @@
+    """Get information about a deployed agent from Galadriel platform."""
+    try:
+        api_key = os.getenv("GALADRIEL_API_KEY")
+        if not api_key:
+            raise click.ClickException("GALADRIEL_API_KEY not found in environment")
+
+        response = requests.get(
+            f"https://api.galadriel.com/agents/{agent_id}",
+            headers={
+                "Content-Type": "application/json",
+                "Authorization": f"Bearer {api_key}",
+            },
+        )
+
+        if not response.status_code == 200:
+            click.echo(
+                f"Failed to get agent state with status {response.status_code}: {response.text}"
+            )
+        click.echo(json.dumps(response.json(), indent=2))
+    except Exception as e:
+        click.echo(f"Failed to get agent state: {str(e)}")
+
+
+@click.command()
+def get_all_agent_states():
+    """Get all agent states"""
+    try:
+        api_key = os.getenv("GALADRIEL_API_KEY")
+        if not api_key:
+            raise click.ClickException("GALADRIEL_API_KEY not found in environment")
+
+        response = requests.get(
+            f"https://api.galadriel.com/agents/",
+            headers={
+                "Content-Type": "application/json",
+                "Authorization": f"Bearer {api_key}",
+            },
+        )
+
+        if not response.status_code == 200:
+            click.echo(
+                f"Failed to get agent state with status {response.status_code}: {response.text}"
+            )
+        click.echo(json.dumps(response.json(), indent=2))
+    except Exception as e:
+        click.echo(f"Failed to get agent state: {str(e)}")
+
+
+def _assert_config_files(image_name: str) -> Tuple[str, str]:
+    if not os.path.exists("docker-compose.yml"):
+        raise click.ClickException("No docker-compose.yml found in current directory")
+    if not os.path.exists(".env"):
+        raise click.ClickException("No .env file found in current directory")
+
+    load_dotenv(dotenv_path=Path(".") / ".env")
+    docker_username = os.getenv("DOCKER_USERNAME")
+    docker_password = os.getenv("DOCKER_PASSWORD")
+    os.environ["IMAGE_NAME"] = image_name  # required for docker-compose.yml
+    if not docker_username or not docker_password:
+        raise click.ClickException(
+            "DOCKER_USERNAME or DOCKER_PASSWORD not found in .env file"
+        )
+    return docker_username, docker_password
+
+
+def _create_agent_template(
+    agent_name: str, docker_username: str, docker_password: str, galadriel_api_key: str
+) -> None:
+    """
+    Generates the Python code and directory structure for a new Galadriel agent.
+
+    Args:
+        agent_name: The name of the agent (e.g., "my_daige").
+    """
+
+    # Create directories
+    agent_dir = os.path.join(agent_name, "agent")
+    agent_configurator_dir = os.path.join(agent_name, "agent_configurator")
+    docker_dir = os.path.join(agent_name, "docker")
+    os.makedirs(agent_dir, exist_ok=True)
```

Comment:
Maybe even remove the `exist_ok=True` here? Dont want to overwrite an existing agent probably?

Reply:
Also couldnt we copy a whole directory somewhere instead of creating a bunch of different dirs? Might be easier to manage in the long run as well?

Reply:
We will likely have the folder somewhere (`project-template` thing) and then inside files use some templating mechanism which allows placeholders like `class {{agent_name}}(GaladrielAgent)` etc. Not sure which tools would be good for this though. This has lower prio IMO than making sure the happy path works.